src/test/java/seedu/dengue/logic/commands/CommandTestUtil.java

```diff
@@ -45,14 +45,14 @@ public class CommandTestUtil {
     public static final String DATE_DESC_BOB = " " + PREFIX_DATE + VALID_DATE_BOB;
     public static final String AGE_DESC_AMY = " " + PREFIX_AGE + VALID_AGE_AMY;
     public static final String AGE_DESC_BOB = " " + PREFIX_AGE + VALID_AGE_BOB;
-    public static final String TAG_DESC_FRIEND = " " + PREFIX_TAG + VALID_TAG_FRIEND;
-    public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
+    public static final String TAG_DESC_DENV2 = " " + PREFIX_TAG + VALID_TAG_DENV2;
+    public static final String TAG_DESC_DENV1 = " " + PREFIX_TAG + VALID_TAG_DENV1;
 
     public static final String INVALID_NAME_DESC = " " + PREFIX_NAME + "James&"; // '&' not allowed in names
     public static final String INVALID_POSTAL_DESC = " " + PREFIX_POSTAL + "911a"; // 'a' not allowed in postals
     public static final String INVALID_DATE_DESC = " " + PREFIX_DATE + "bob!yahoo"; // missing '@' symbol
     public static final String INVALID_AGE_DESC = " " + PREFIX_AGE; // empty string not allowed for age
-    public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "hubby*"; // '*' not allowed in tags
+    public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "sever"; // "severe" not allowed in tags
```

Comment:
Should the comment be changed to more accurately reflect why this tag is invalid? e.g. `// "sever" is not an accepted tag`

Reply:
Good call!

```suggestion
    public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "severe"; // "severe" is not a valid tag
```